data-serving/scripts/export-data/test_transform_mongoexport_expected.json

```diff
@@ -0,0 +1,3 @@
_id": "ObjectId(798989a98998acc98989a1bb)", "caseReference.additionalSources": null, "caseReference.sourceEntryId": "", "caseReference.sourceId": "787123878aa90909811aaff1", "caseReference.sourceUrl": "http://foo/bar.csv", "caseReference.uploadIds": "bb12399abbb19230900aa123", "caseReference.verificationStatus": "UNVERIFIED", "demographics.ageRange.end": "29", "demographics.ageRange.start": "20", "demographics.ethnicity": "", "demographics.gender": "Female", "demographics.nationalities": null, "demographics.occupation": "", "events.confirmed.date": "2021-01-05", "events.confirmed.value": null, "events.firstClinicalConsultation.date": "", "events.hospitalAdmission.date": "", "events.hospitalAdmission.value": "", "events.icuAdmission.date": "", "events.icuAdmission.value": "", "events.onsetSymptoms.date": "", "events.outcome.date": "", "events.outcome.value": "", "events.selfIsolation.date": "", "genomeSequences": "[]", "genomeSequences.repositoryUrl": "", "genomeSequences.sampleCollectionDate": "", "genomeSequences.sequenceId": "", "genomeSequences.sequenceLength": "", "genomeSequences.sequenceName": "", "location.administrativeAreaLevel1": "", "location.administrativeAreaLevel2": "", "location.administrativeAreaLevel3": "", "location.country": "Antarctica", "location.geoResolution": "Country", "location.geometry.latitude": "-79.402", "location.geometry.longitude": "0.323", "location.name": "Antarctica", "location.place": "", "notes": "", "pathogens": null, "preexistingConditions.hasPreexistingConditions": "", "preexistingConditions.values": "", "revisionMetadata.creationMetadata.date": "2021-01-02T13:42:34.991Z", "revisionMetadata.creationMetadata.notes": "", "revisionMetadata.editMetadata.date": "", "revisionMetadata.editMetadata.notes": "", "revisionMetadata.revisionNumber": "0", "symptoms.status": "", "symptoms.values": "", "transmission.linkedCaseIds": "", "transmission.places": "", "transmission.routes": "", "travelHistory.travel.dateRange.end": "", "travelHistory.travel.dateRange.start": "", "travelHistory.travel.location.administrativeAreaLevel1": "", "travelHistory.travel.location.administrativeAreaLevel2": "", "travelHistory.travel.location.administrativeAreaLevel3": "", "travelHistory.travel.location.country": "", "travelHistory.travel.location.geoResolution": "", "travelHistory.travel.location.geometry.coordinates": "", "travelHistory.travel.location.name": "", "travelHistory.travel.location.place": "", "travelHistory.travel.methods": "", "travelHistory.travel.purpose": "true", "travelHistory.traveledPrior30Days": "", "vaccines.0.batch": "", "vaccines.0.date": "", "vaccines.0.name": "", "vaccines.0.sideEffects": "", "vaccines.1.batch": "", "vaccines.1.date": "", "vaccines.1.name": "", "vaccines.1.sideEffects": "", "vaccines.2.batch": "", "vaccines.2.date": "", "vaccines.2.name": "", "vaccines.2.sideEffects": "", "vaccines.3.batch": "", "vaccines.3.date": "", "vaccines.3.name": "", "vaccines.3.sideEffects": null, "variantOfConcern": ""}
```

Comment:
This needs a closing `]`.

Reply:
`genomeSequences` should not be present.

Reply:
Are we removing genomeSequences entirely from the output? Then I'll update the data dictionary as well.

Reply:
Yeah, that was discussed and changed in https://github.com/globaldothealth/list/pull/2355